tests/src/test/scala/outwatch/ScenarioTestSpec.scala

```diff
@@ -235,31 +237,34 @@ class ScenarioTestSpec extends JSDomAsyncSpec {
       clickStream <- Handler.createF[IO, MouseEvent]
       keyStream <- Handler.createF[IO, KeyboardEvent]
 
-      buttonDisabled = textFieldStream
+
+    } yield {
```

Comment:
Why did we rewrite this? :)

Reply:
because of this compiler bug:
https://github.com/lampepfl/dotty/issues/14171